### PR TITLE
JS Packages: Add Base Styles package

### PR DESCRIPTION
**Pull request by kraftbj**
See #20629

Removing elements from 20629 to make it easier to digest what's changing and what may be breaking.

#### Changes proposed in this Pull Request:

* Adds a base-styles packages. Not used yet.

#### Jetpack product discussion

n/a

#### Does this pull request change what data or activity we track or use?

No.

#### Testing instructions:

* None yet. This just adds a shell packages. Nothing to see here.
*

### CODE REVIEW

**Review by gravityrail — APPROVED** [review]
LGTM! Thanks for putting this together.

File: projects/js-packages/base-styles/composer.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+		{
+			"type": "path",
+			"url": "../../packages/*",
+			"options": {
+				"monorepo": true
+			}
+		}
+	],
+	"minimum-stability": "dev",
+	"prefer-stable": true
```
Comment (jeherve) [inline projects/js-packages/base-styles/composer.json:29]: Let's add the other monorepo tools here:

```suggestion
	"prefer-stable": true,
	"extra": {
		"autotagger": true,
		"npmjs-autopublish": true,
		"mirror-repo": "Automattic/jetpack-base-styles",
		"changelogger": {
			"link-template": "https://github.com/Automattic/jetpack-base-styles/compare/${old}...${new}"
		}
	}
```

**Review by jeherve — CHANGES_REQUESTED** [review]

File: projects/js-packages/base-styles/composer.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+		"yoast/phpunit-polyfills": "0.2.0",
+		"automattic/jetpack-changelogger": "^1.2"
+	},
+	"autoload": {
+		"classmap": [
+			"src/"
+		]
+	},
+	"scripts": {
+		"build-development": "echo 'Add your build step to composer.json, please!'"
```
[the remark this answers is not in the document]
Comment (kraftbj) [reply projects/js-packages/base-styles/composer.json]: 705a709

**Review by jeherve — APPROVED** [review]

**Comment by kraftbj** [comment]
r230308-wpcom

**Pull request merged**